File: C++ Projects/Blackjack/Blackjack/Blackjack.cpp
// Blackjack.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include <iostream>
using namespace std;


int main()
{
	int n = 0;
	while (n <= 21) 
	{
		int c;
		cout << "Please enter a number between 1 and 11: " << endl;
		cin >> c;
		cout << c << endl;
		if ((c > 0) && (c < 12))
		{
			n = n + c;
		}
		else 
		{
			cout << "Out of range; rejected" << endl;
		}
	}
	cout << "The total is " << n << endl;
	return 0;
}

File: Practicing-Coding/Blackjack.py
n = 0
while n <= 21:
    print('Please enter a number between 1 and 11: ',end='')
    c = int(input())
    print(c)
    if c <=11:
        n = n + c
    else:
        print('Out of range; rejected.')
        break
    if n > 21:
        print('The total is',n)
        break